.github/workflows/scheduled_test_cpu.yml

```diff
@@ -0,0 +1,52 @@
+name: (scheduled) Run tests on CPU
+
+on:
+  schedule:
+    - cron: "0 4 * * *"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  tests-cpu:
+    name: ${{ matrix.os }} - python-${{ matrix.python-version }}, torch-${{ matrix.pytorch-version }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos]
+        python-version: ['3.7', '3.8']
+        pytorch-version: ['1.9.1', '1.10.2', '1.12.1', '1.13.0', 'nightly']
```

Comment:
i added 1.9 and 1.13 for the scheduled tests

Reply:
perfect